c2corg_ui/templates/waypoint/index.html

```diff
@@ -19,7 +19,7 @@
     from json import dumps, loads
     features = [];
     for waypoint in waypoints:
-        if waypoint['geometry']['geom'] is not None:
+        if waypoint.get('geometry') and waypoint.get('geometry').get('geom'):
```

Comment:
Why do you need to do this in a 404 page PR?
Is it really related? If not please move the commit in a separated PR.